TVs/Thomson/Thomson_UnknownModel1.ir

```diff
@@ -0,0 +1,38 @@
+Filetype: IR signals file
+Version: 1
+# 
+name: POWER
+type: raw
+frequency: 38000
+duty_cycle: 0.330000
+data: 4008 3984 520 1976 517 1980 523 1975 518 1979 514 1016 492 1012 496 1975 518 987 521 1976 517 987 521 1977 516 1014 494 1010 488 990 518 985 523 1008 489 1981 522 1976 517 1013 495 1976 517 987 521 1977 516 987 521 1977 516 8473 4002 3988 516 1979 514 1984 519 1977 516 1982 521 1008 489 988 520 1977 516 988 520 1977 516 988 520 1977 516 987 521 1010 488 989 519 985 523 980 518 2006 487 1984 519 983 514 1983 520 983 515 1983 520 983 515 1982 521 8466 4010 3978 516 1980 523 1974 519 1978 578 1918 522 982 516 1014 494 1977 516 987 521 1976 517 987 521 1976 517 986 522 982 515 1015 493 1011 487 990 518 1979 514 1984 519 984 524 1973 520 984 513 1984 519 983 514 1983 520
```

Comment:
which is the shortest possible data package?

Reply:
It varies by command. If you run the IR files through the [Pulse Plotter](https://my.flipp.dev/pulse-plotter) you can get a better idea of what's going on.